### PR TITLE
Update styles of releases table

### LINKED ISSUE #1187: Implement updated visual design

### PR DESCRIPTION
**Pull request by bartaz**
Fixes #1187 

Updates visual style of releases table on Release page.
Design doc: https://docs.google.com/document/d/1DdWKGN22mppdse3jNmQe1p02ebimFO1ww94BXHD9Byg/edit?ts=5bab92e3#

### QA
- ./run or http://snapcraft.io-canonical-websites-pr-1206.run.demo.haus/
- go to Releases page of any snap
- look at the release table (it should have new look)
- select some revisions to release
- promote some channels
- close some channels
- check some snap with devmode revisions
- all should work as before with new look

screen shot 2018-10-17 at 15 46 20

### CODE REVIEW

File: static/js/publisher/release/revisionsTable.js [hunk 1]
```diff
@@ -307,29 +311,25 @@ export default class RevisionsTable extends Component {
   }
 
   render() {
-    const { releasedChannels, archs, tracks } = this.props;
-
+    let { releasedChannels, archs, tracks } = this.props;
+    //archs = archs.concat(archs).concat(archs).concat(['test']);
```
Comment (Lukewh) [inline static/js/publisher/release/revisionsTable.js]: should probably remove this if it was for testing while dev'ing.

Comment (bartaz) [reply static/js/publisher/release/revisionsTable.js]: Damn, every time I was explicitly skipping it from the commit I knew that there will be this one time when I'm gonna commit it.

**Comment by therealjuan** [comment]
Everything works from a QA standpoint but we will discuss in a meeting the UX side of things.

**Comment by bartaz** [comment]
@therealjuan Can you have a look and give it a UX review when you have a moment?

**Comment by therealjuan** [comment]
UX reviewed and we already agreed next steps so 👍

**Pull request merged**